src/components/inputs/localized-rich-text-input/editor.js

```diff
@@ -0,0 +1,240 @@
+              key={props.language}
+              css={css`
+                width: 100%;
+                position: relative;
+                display: flex;
+              `}
+            >
+              <label
+                htmlFor={props.id}
+                css={theme => getLanguageLabelStyles(props, theme)}
+              >
+                {/* FIXME: add proper tone for disabled when tones are refactored */}
+                <Text.Detail tone="secondary">
+                  {props.language.toUpperCase()}
+                </Text.Detail>
+              </label>
+
+              <RichTextBody
+                ref={{
+                  containerRef: ref,
+                  registerContentNode,
+                }}
+                styles={{
+                  container: css`
+                    flex: auto;
+                    width: 100%;
+                    border-top-left-radius: 0;
+                    border-bottom-left-radius: 0;
+                  `,
+                }}
+                isOpen={props.isOpen}
+                hasError={props.hasError}
+                isDisabled={props.isDisabled}
+                hasWarning={props.hasWarning}
+                isReadOnly={props.isReadOnly}
+                editor={props.editor}
+                containerStyles={containerStyles}
+                showExpandIcon={props.showExpandIcon}
+                onClickExpand={props.onClickExpand}
+              >
+                {props.children}
+              </RichTextBody>
+            </div>
+            <Row>
+              {(() => {
+                if (props.error)
+                  return (
+                    <LeftColumn>
+                      <div>{props.error}</div>
+                    </LeftColumn>
+                  );
+                if (props.warning)
+                  return (
+                    <LeftColumn>
+                      <div>{props.warning}</div>
+                    </LeftColumn>
+                  );
+                return (
+                  languagesControl && (
+                    <LeftColumn>{languagesControl}</LeftColumn>
+                  )
+                );
+              })()}
+              {renderToggleButton && isOpen && (
+                <React.Fragment>
+                  <LeftColumn />
+                  <RightColumn>
+                    <FlatButton
+                      onClick={toggle}
+                      label={intl.formatMessage(messages.collapse)}
+                      icon={<AngleUpIcon size="small" />}
+                    />
+                  </RightColumn>
+                </React.Fragment>
+              )}
+            </Row>
+            {(props.error || props.warning) && props.languagesControl()}
+          </Spacings.Stack>
+        );
+      }}
+    </CollapsibleMotion>
+  );
+};
+
+// eslint-disable-next-line react/display-name
+const renderEditor = (props, editor, next) => {
+  const children = React.cloneElement(next(), {
+    tagName: 'output',
```

Comment:
same as in `RichTextInput`. We use `output` because it supports labels with `htmlFor` while "div" (the default) does not.